sdk/vc-data/src/data/experience/v1.ts

```diff
@@ -0,0 +1,58 @@
+import { VCV1, VCV1Subject } from '@affinidi/vc-common'
+
+import { PersonEV1, getBaseV1ContextEntries } from '../base/v1'
+import {
+  CreateThing,
+  MaybeArray,
+  ExtendThing,
+  Type,
+  ExpandThing,
+  createContextEntry,
+  createVCContextEntry,
+} from '../util'
+
+type ExperiencePersonV1Mixin = CreateThing<
+  'ExperiencePerson',
+  {
+    name: string
+    officialEmail: string
+    organizationName: string
+    titlesHeld: MaybeArray<string>
+    employmentType: 'Full-time' | 'Part-time' | 'Contractor' | 'Intern'
+    dateOfJoining: string
+    employeeID?: string
+    personalEmail?: string
+    phoneNumber?: string
+    EmployeeAddress?: string
```

Comment:
@sridharan21994 Why its start from capital character ?

Reply:
fixed this with new commit